src/commands.ts

```diff
@@ -659,6 +696,11 @@ export const insertList =
       if (!dispatch) {
         return true
       }
+
+      if (list.node.attrs.listStyleType === style) {
```

Comment:
Are we sure that's a condition precise enough to split the list?

Reply:
update method name to `toggleOffList` as in this case we should remove all list items and return a separate paragraphs for each list item